### PR TITLE
Completed issue #10 Build training page hero

### PR DESCRIPTION
**Pull request by adityayadav71**
1. Created training.astro file in pages.
2. Created TrainingHero.tsx and TrainingHeroCarousel.tsx components.
3. Added styles.css file to support custom swiper navigation buttons.
4. Added relevant image and icon resources in public folder.

### CODE REVIEW

**Review by demonkillerr — COMMENTED** [review]
code looks good to me

**Review by vish-rt — COMMENTED** [review]
LGTM

**Review by demonkillerr — DISMISSED** [review]
please go through image optimization guides in resources' tab and make changes as request by @JayeshVP24

File: public/book-open.svg
Comment (JayeshVP24) [inline public/book-open.svg:1]: move svgs to misc folder

File: public/student-workshop.png
Comment (JayeshVP24) [inline public/student-workshop.png:1]: move to events folder in public dir

File: src/components/TrainingHeroCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      slidesPerView={1}
+      navigation={{
+        prevEl: ".swiper-prev",
+        nextEl: ".swiper-next",
+        disabledClass: "swiper-button-disabled",
+      }}
+      className="h-full relative"
+    >
+      <SwiperSlide className="relative">
+        <img src="/student-workshop.png" alt="student-workshop" className="h-full w-auto object-cover rounded-xl" />
```
Comment (JayeshVP24) [inline src/components/TrainingHeroCarousel.tsx]: use `<Image>` tag of @astro everywhere

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by riyavishwa981 — APPROVED** [review]

**Review by JayeshVP24 — APPROVED** [review]

**Pull request merged**